src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -316,10 +322,18 @@ public PravegaStorageDriver(
 		super(stepId, dataInput, storageConfig, verifyFlag, BYTES.equals(streamDataType) ? 1 : batchSize);
 		this.concurrencyThrottle = new Semaphore(concurrencyLimit > 0 ? concurrencyLimit : Integer.MAX_VALUE, true);
 		val driverConfig = storageConfig.configVal("driver");
+		val createConfig = driverConfig.configVal("create");
+		this.recordWriteTimeFlag = createConfig.boolVal("timestamp");
+		if (recordWriteTimeFlag) {
+			Loggers.ERR.warn("{}: Make sure that event size is not smaller than 8 bytes ", stepId);
```

Comment:
Could you, please, explain in a high level, what'd happen it the event size would be smaller than 8 bytes?
If things would be broken, maybe it'd be more sufficient to throw an exception or change log level to error?

Reply:
Mongoose would crash on write operation. Throwing an exception would be nice but the size of event is controlled by `item-data-size` which is handled in mongoose-base and we don't know its value in driver and we shouldn't know. I'll raise level of this log.